packages/salesforcedx-vscode-core/src/commands/forceApexTestRun.ts

```diff
@@ -91,6 +91,7 @@ export class ForceApexTestRunExecutor extends SfdxCommandletExecutor<
         .withArg('force:apex:test:run')
         .withFlag('--suitenames', `${data.label}`)
         .withFlag('--resultformat', 'human')
+        .withFlag('--loglevel', 'error')
```

Comment:
Is this to prevent the warning on the prompt?

`{"warnings":["apiVersion configuration overridden at 42.0"]}`

Does this happen frequently? I'm asking because we should probably fix this at the `CommandExecutor` level to just default loglevel to error so that we don't have to duplicate this for every command.

Reply:
Leave this one as-is. We will fix this in a later PR as part of  W-4485495

Reply:
Copy that.